### PR TITLE
Small changes in README.md

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,6 @@ Graphql\init(new \GraphQL\Schema(['query' => $query(), 'mutation' => $mutation()
 ---
 More action here: [siler-examples](https://github.com/leocavalcante/siler-examples)
 
-⚠️️ **This is a work in progress, API may change** 🚧<br>
```
Comment (leocavalcante) [inline README.md]: Thank you very much for your corrections.
Can I ask you only to maintain this break lines? Without them the footer get one-lined.
Cheers.

File: README.md [hunk 2]
```diff
@@ -156,6 +156,6 @@ Graphql\init(new \GraphQL\Schema(['query' => $query(), 'mutation' => $mutation()
...
-But if you give a try, I'd love the get some feedback<br>
```
Comment (leocavalcante) [inline README.md]: An here too, then MIT 2017 get its own line.

Comment (lex111) [inline README.md]: @leocavalcante Oops, sorry, it really looks bad, missed out of sight, added blank lines instead of br tag.

**Comment by leocavalcante** [comment]
Thank you!

**Pull request merged**